src/cosmoz-treenode-navigator.ts

```diff
@@ -70,6 +71,11 @@ const NodeNavigator = ({
 		'highlightedNode',
 	);
 
+	const [nodesOnNodePath, setNodesOnNodePath] = useProperty<Node[]>(
+		'nodesOnNodePath',
+		initialNodesOnNodePath || [],
+	);
+
```

Comment:
`useProperty` takes it's initial value from `host[property]`, so you don't need to do it yourself here.

```suggestion
	const [nodesOnNodePath, setNodesOnNodePath] = useProperty<Node[]>(
		'nodesOnNodePath',
		[],
	);

```